scanpy/plotting/_utils.py

```diff
@@ -482,14 +483,18 @@ def add_colors_for_categorical_sample_annotation(
         _set_default_colors_for_categorical_obs(adata, key)
 
 
-def plot_edges(axs, adata, basis, edges_width, edges_color):
+def plot_edges(axs, adata, basis, edges_width, edges_color, neighbors_key=None):
```

Comment:
Like with the leiden function, I think that if only one of the graphs is needed, we could just pass the `obsp` key to that graph.

Reply:
Added obs_key to leiden, louvain, draw_graph as discussed.

Reply:
Could it just be `obsp`? I think it's a little more consistent with the `layer` argument we use in a few places.

Reply:
Ok, you are right.

Reply:
Great, thanks!